src/couch/rebar.config.script

```diff
@@ -191,7 +196,7 @@ IcuDarwinEnv = [{"CFLAGS", "-DXP_UNIX -I/usr/local/opt/icu4c/include -I/opt/home
 IcuBsdEnv = [{"CFLAGS", "-DXP_UNIX -I/usr/local/include"},
              {"LDFLAGS", "-L/usr/local/lib"}].
 IcuWinEnv = [{"CFLAGS", "$DRV_CFLAGS /DXP_WIN"},
-             {"LDFLAGS", "icuin.lib icudt.lib icuuc.lib"}].
+             {"LDFLAGS", "$LDFLAGS icuin.lib icudt.lib icuuc.lib"}].
```

Comment:
Should this one be inheriting `$DRV_LDFLAGS` for consistency?

Reply:
Here I think the $DRV_CFLAGS is a now harmless redundancy (AFAICT the template for a C in a DRV context be combining $CFLAGS $DRV_CFLAGS again so it is equivalent to $DRV_CFLAGS $CFLAGS $DRV_CFLAGS) rather than achieving the same thing as $CFLAGS would, my guesses for its existence are:
  - could fix something in terms of order
  - was maybe important in some version of rebar2? 

I think this comes back to comment 1-(1) and whether $CFLAGS support should be added in windows for equivalence even though %INCLUDE% covers everything currently being set.

Reply:
I should probably note that the 3 times an EXE_FLAGS is used in compiles it is generating warnings consistent with using this default [EXE_FLAGS](https://github.com/blt/port_compiler/blob/master/src/pc_port_env.erl#L240) while [win32 has DRV_CFLAGS overrides by default](https://github.com/blt/port_compiler/blob/master/src/pc_port_env.erl#L305) which makes a clean compile, that's how I surmised that I don't want to override rebar's DRV_CFLAGS.

Reply:
Thanks Will.